src/entry-point.tsx

```diff
@@ -166,4 +171,19 @@ export class LibraryController {
             this.refreshLibraryViewHandler();
         }
     }
+
+    modifyLibraryItemData(context:any ,data:ItemData):boolean{
+        if(this.modifyItemHandler){
+            return this.modifyItemHandler(context,data);
+        }
+       return false;
+    }
+    //TODO summary
+    setHostContext(context:HostingContextType):void{
+        console.log("I AM HERE")
```

Comment:
Where are you?

Reply:
hah, will remove. This reminds me, I had really bad luck trying to debug the jest tests - have you or @EnzoBatistaU been able to get that working?